resources/js/components/basic-select-options.tsx

```diff
@@ -32,6 +32,8 @@ export default class RankingSelectOptions extends React.PureComponent<Props> {
 
   private href(id: number | null) {
     switch (this.props.type) {
+      case 'judge_results':
+        return route('contest-entries.judge-results', { contest_entry: id });
```

Comment:
oh yeah, `contest_entry` needs to be a non-nullable value

Reply:
how would you go about resolving that? ...honestly not really ceratin about what to do with it🤔

Reply:
just set it to 0 or return an empty href for now